webpack/ForemanInventoryUpload/SubscriptionsPageExtension/InventoryAutoUpload/index.js

```diff
@@ -0,0 +1,22 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from '../../Components/AutoUploadSwitcher/AutoUploadSwitcherActions';
+import InventoryAutoUpload from './InventoryAutoUpload';
+import { selectAutoUploadEnabled } from '../../Components/AccountList/AccountListSelectors';
+import { fetchAccountsStatus as fetchSettings } from '../../Components/AccountList/AccountListActions';
+
+// map state to props
+const mapStateToProps = state => ({
+  autoUploadEnabled: selectAutoUploadEnabled(state),
+});
+
+// map action dispatchers to props
+const mapDispatchToProps = dispatch =>
+  bindActionCreators({ ...actions, fetchSettings }, dispatch);
```

Comment:
Can you use `useDispatch` and `useSelector` here?

Reply:
We prefer to keep it with the connect convention mostly for easier cherry-picks,
also looks like except for the syntactic sugar hooks provides, performance-wise it might be less efficient,
https://itnext.io/how-existing-redux-patterns-compare-to-the-new-redux-hooks-b56134c650d2

Reply:
Sure, thanks for the explanation!